src/status_im2/contexts/syncing/setup_syncing/view.cljs

```diff
@@ -38,31 +38,35 @@
 
 (defn view
   []
-  (let [profile-color (rf/sub [:profile/customization-color])
-        valid-for-ms  (reagent/atom code-valid-for-ms)
-        code          (reagent/atom nil)
-        delay         (reagent/atom nil)
-        timestamp     (reagent/atom nil)
-        set-code      (fn [connection-string]
-                        (when (sync-utils/valid-connection-string? connection-string)
-                          (reset! timestamp (* 1000 (js/Math.ceil (/ (datetime/timestamp) 1000))))
-                          (reset! delay 1000)
-                          (reset! code connection-string)))
-        clock         (fn []
-                        (if (pos? (- code-valid-for-ms
-                                     (- (* 1000 (js/Math.ceil (/ (datetime/timestamp) 1000)))
-                                        @timestamp)))
-                          (swap! valid-for-ms (fn [_]
-                                                (- code-valid-for-ms
-                                                   (- (* 1000
-                                                         (js/Math.ceil (/ (datetime/timestamp) 1000)))
-                                                      @timestamp))))
-                          (reset! delay nil)))
+  (let [{:keys [key-uid customization-color]} (rf/sub [:profile/multiaccount])
+        valid-for-ms (reagent/atom code-valid-for-ms)
+        code (reagent/atom nil)
+        delay (reagent/atom nil)
+        timestamp (reagent/atom nil)
+        set-code (fn [connection-string]
+                   (when (sync-utils/valid-connection-string? connection-string)
+                     (reset! timestamp (* 1000 (js/Math.ceil (/ (datetime/timestamp) 1000))))
+                     (reset! delay 1000)
+                     (reset! code connection-string)))
+        clock (fn []
+                (if (pos? (- code-valid-for-ms
+                             (- (* 1000 (js/Math.ceil (/ (datetime/timestamp) 1000)))
+                                @timestamp)))
+                  (swap! valid-for-ms (fn [_]
+                                        (- code-valid-for-ms
+                                           (- (* 1000
+                                                 (js/Math.ceil (/ (datetime/timestamp) 1000)))
+                                              @timestamp))))
+                  (reset! delay nil)))
         cleanup-clock (fn []
                         (reset! code nil)
                         (reset! timestamp nil)
-                        (reset! valid-for-ms code-valid-for-ms))]
-
+                        (reset! valid-for-ms code-valid-for-ms))
+        on-enter-password (fn [password]
+                            (rf/dispatch [:set-in [:profile/login :key-uid] key-uid])
+                            (rf/dispatch
+                             [:syncing/get-connection-string-for-bootstrapping-another-device
+                              password set-code]))]
     (fn []
```

Comment:
https://github.com/reagent-project/reagent/blob/a14faba55e373000f8f93edfcfce0d1222f7e71a/doc/CreatingReagentComponents.md?plain=1#L153

Reply:
but the outer function here also has no params? 🤔

Reply:
<img width="124" alt="Screenshot 2023-08-18 at 11 50 27" src="https://github.com/status-im/status-mobile/assets/22799766/667475c1-88e1-4522-a21a-4c06c1b6fefc">

Reply:
You are 100% correct. My bad :)